### ISSUE TITLE
Actions are triggered only on the second touch

### ISSUE THREAD

**Issue by PoisonousJohn** [qmlos/shell#110]
On my laptop Lenovo X1 Yoga, if I touch any icon (battery, clock, etc), panel is opened only from the second touch.

I don't have such problem if I click with mouse.

The same problem goes with app launch icons.

**Comment by plfiorini** [comment]
@PoisonousJohn Does it happen with windows too?

Also, open the menu (Super+Space or click the first button in the panel), switch to the categories mode. Can you select a category with a single tap?

**Comment by PoisonousJohn** [comment]
> Can you select a category with a single tap?

No
> Does it happen with windows too?

Windows are activated with a single tap.

P.S. I should say that I'm running with -platform xcb

**Comment by plfiorini** [comment]
@PoisonousJohn How does it work on tty?

**Comment by plfiorini** [comment]
And... is there any difference in behavior between touch screen and touch pad?

**Comment by PoisonousJohn** [comment]
Yes, when I talked that mouse click works fine, I meant a click using
TouchPad. So only touch screen has incorrect behavior

3 окт. 2017 г. 12:40 пользователь "Pier Luigi Fiorini" <
notifications@github.com> написал:

**Comment by plfiorini** [comment]
@PoisonousJohn Can you print what `sudo libinput list-devices` says? And also capture the output of `sudo libinput debug-events` when single tapping so we can see what libinput events are generated.
